src/librustc_trans/trans/common.rs

```diff
@@ -114,23 +114,24 @@ pub fn normalize_ty<'tcx>(cx: &ty::ctxt<'tcx>, ty: Ty<'tcx>) -> Ty<'tcx> {
 }
 
 // Is the type's representation size known at compile time?
-pub fn type_is_sized<'tcx>(cx: &ty::ctxt<'tcx>, ty: Ty<'tcx>) -> bool {
-ty::type_contents(cx, ty).is_sized(cx)
+pub fn type_is_sized<'a,'tcx>(ccx: &CrateContext<'a,'tcx>, ty: Ty<'tcx>) -> bool {
```

Comment:
Changing this fn and lltype_is_sized to take a CrateCtxt rather than a ty::ctxt seems unnecessary and makes things more complicated.